lib/gap_intelligence/client/requestable.rb

```diff
@@ -8,21 +8,21 @@ module Requestable
     # @yield [req] The Faraday request
     def perform_request(method, path, options = {}, &block)
       record_class = options.delete(:record_class)
+      response_processor = options.delete(:response_processor)
```

Comment:
@gtimti thanks for your comments, i'm changing the way how api returns data so we could process data as Hash and do not change anything here